docs/docs/customize/layout-engines.md

```diff
@@ -100,3 +100,9 @@ context.PluginManager.AddPlugin(treeLayoutPlugin);
 ```
 
 ![TreeLayoutEngine demo](../../images/tree-layout-demo.gif)
+
+### `FreeLayoutEngine`
+
+<xref:Whim.Layout.FreeEngineLayout> is a layout that allows users to have only free-floating windows.
+
+![FreeLayoutEngine demo](../../images/free-layout-demo.gif)
```

Comment:
I didn't do the gif.
I don't know if you want to add one.
But if I add it, it will different style from the already existing one.

Reply:
A gif would be nice. No worries about the styling - my desktop doesn't look like that anymore anyway.

In case it helps, the tool/site I use to capture gifs is https://gifcap.dev/.

Reply:
Ok, I will add it at the end, when all bugs are solved.

Reply:
Done ~ But not very appealing

Reply:
No worries, thanks!